hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneTenantShell.java

```diff
@@ -558,10 +558,17 @@ public void testOzoneTenantBasicOperations() throws IOException {
         "user", "assign", "bob", "--tenant=research",
         "--accessId=research$bob42"});
     checkOutput(out, "", false);
+    // HDDS-6366: Disallow specifying custom accessId.
+    checkOutput(err, "Failed to assign 'bob' to 'research': "
+        + "Invalid accessId 'research$bob42'. "
+        + "Specifying a custom access ID disallowed. "
+        + "Expected accessId to be assigned is 'research$bob'\n", true);
```

Comment:
How about adding a test assigning bob again, to make sure that fails since he is already added? That would be kind of a replacement for the commented out test.

Reply:
Added. Though the error message is different.